res/css/views/right_panel/_ExtensionsCard.pcss

```diff
@@ -130,4 +128,9 @@ Please see LICENSE files in the repository root for full details.
         /* Overlap the Add extensions button */
         top: -76px;
     }
+
+    .mx_EmptyState {
+        /* Stop empty state scrolling */
+        height: calc(100% - 76px);
```

Comment:
Maybe declare a CSS variable with `-76px` value and use it in the `_ExtensionsCard.pcss` file